config/runtime.exs

```diff
@@ -12,6 +12,7 @@ external_url = System.get_env("EXTERNAL_URL", "http://localhost:4000")
 
 # Enable Forwarded headers, e.g 'X-FORWARDED-HOST'
 proxy_forwarded = FzString.to_boolean(System.get_env("PROXY_FORWARDED") || "false")
+trusted_proxy = FzString.to_array(System.get_env("TRUSTED_PROXY") || "[]")
```

Comment:
Is this a comma separated list or a square bracket wrapped list? FzString.to_array does not seem to be able to handle `[]`

Reply:
Will fix it later, haven't fixed all the details in this PR yet, when loading trusted_proxy we should check it's a list of valid ip ranges.